packages/mobile-app/.env.template

```diff
@@ -1 +0,0 @@
-EXPO_PUBLIC_DEMO_API=true
```

Comment:
Is this supposed to be deleted? Usually we use it as an example file that can be copied to `.env`